### PR TITLE
Dzhupyna volodymyr

### PR DESCRIPTION
**Pull request by kolyasalubov**

### CODE REVIEW

File: HW1/vdzhup/1.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+user_name = input("What is your name? ")
+print("Hello", user_name)
+
+user_age = (input("How old are you? "))
+print("Your age is", user_age)
+
+user_location = input("Where do you live? ")
+print("You live in", user_location)
```
Comment (kolyasalubov) [inline HW1/vdzhup/1.py:8]: good job

File: HW1/vdzhup/2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+first_number = 10
+second_number = 3
+
+print("a + b =", first_number + second_number)
+print("a - b =", first_number - second_number)
+print("a * b =", first_number * second_number)
+print("a / b =", first_number / second_number)
+print("a // b =", first_number // second_number)
+print("a ** b =", first_number ** second_number)
+print("a % b =", first_number % second_number)
```
Comment (kolyasalubov) [inline HW1/vdzhup/2.py:10]: good job

File: HW3/vdzhup/1.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+print("'better' count:", count_better)
+print("'never' count:", count_never)
+print("'is' count:", count_is)
+
+upper_text = python_philosofy.upper()
+print(upper_text)
+
+replace_text = python_philosofy.replace("i", "&")
+print(replace_text)
```
Comment (kolyasalubov) [inline HW3/vdzhup/1.py:33]: good job

File: HW3/vdzhup/2.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+multiplication = first_digit * second_digit * \
+                third_digit * fourth_digit
+print(f"Product of digits: {multiplication}")
+
+reverse_oder_digits = int(str(user_num)[::-1])
+print(f"Reverse order of digits: {reverse_oder_digits}")
+
+increase_sorted_num = int("".join(sorted(str(user_num))))
+print(f"Sorting by increasing: {increase_sorted_num}")
```
Comment (kolyasalubov) [inline HW3/vdzhup/2.py:16]: excellent job

File: HW3/vdzhup/3.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+first_number = 5
+second_number = 10
+print("first_number -", first_number, ";", "second_number -", second_number)
+
+first_number, second_number = second_number, first_number
+print("first_number -", first_number, ";", "second_number -", second_number)
```
Comment (kolyasalubov) [inline HW3/vdzhup/3.py:6]: good job

File: HW4/vdzhup/1.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+factorial_count = 1
+
+if user_factorial < 0:
+    print(f"{user_factorial} is non positive number!")
+else:
+    while user_factorial > 1:
+        factorial_count *= user_factorial
+        user_factorial -= 1
+    print(f"The factorial of the number {user_factorial}! is: {factorial_count}")
+
```
Comment (kolyasalubov) [inline HW4/vdzhup/1.py:14]: good job
what will be in case user_factorial=1?

File: HW4/vdzhup/1.py [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+user__factorial = int(input("Enter only the positive number for the factorial: "))
+
+factorial__count = 1
+
+if user__factorial < 0:
+    print(f"{user__factorial} is non positive number!")   
+else:
+    for i in range(1, user__factorial + 1):
+        factorial__count *= i
+    print(f"The factorial of the number {user__factorial}! is: {factorial__count}")
```
Comment (kolyasalubov) [inline HW4/vdzhup/1.py:28]: good job
consider another way

input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

File: HW5/vdzhup/1.py [hunk 8]
```diff
@@ -0,0 +1,6 @@
+user_count_of_number = int(input("How many numbers you want to enter: "))
+
+integer_list = [int(input(f"Enter your {item + 1} number: ")) for item in range(user_count_of_number)]
+float_list = [float(item) for item in integer_list]
+
+print(f"Before change: {integer_list}, after change: {float_list}")
```
Comment (kolyasalubov) [inline HW5/vdzhup/1.py:6]: good job

File: HW5/vdzhup/2.py [hunk 9]
```diff
@@ -0,0 +1,10 @@
+user_fibonacci_number = int(input("Enter the number to create a Fibonacci sequence: "))
+
+fibonacci_list = []
+sequnce_result = 1    # 0 1 1 2 3 5 8
+count_of_sequence = 0 # 1 1 2 3 5 8 13
+
+for i in range(user_fibonacci_number + 1):
+    sequnce_result, count_of_sequence = count_of_sequence, sequnce_result + count_of_sequence
+    fibonacci_list.append(sequnce_result)
+print(f"Number {user_fibonacci_number} creates the following sequence: {fibonacci_list}")
```
Comment (kolyasalubov) [inline HW5/vdzhup/2.py:10]: good job

File: HW6/vdzhup/1.py [hunk 10]
```diff
@@ -0,0 +1,8 @@
+first_list = [item for item in range(1, 10) if item % 2 == 0]
+print(f"Even numbers that are divisible by 2 in range from 1 to 10: {first_list}")
+
+second_list = [item for item in range(1, 10) if item % 2 != 0 and item % 3 == 0]
+print(f"Odd numbers, which are divisible by 3 in range from 1 to 10: {second_list}")
+
+third_list = [item for item in range(1, 10) if item % 2 != 0 and item % 3 != 0]
+print(f"Numbers that are not divisible by 2 and 3 in range from 1 to 10: {third_list}")
```
Comment (kolyasalubov) [inline HW6/vdzhup/1.py:8]: good job

File: HW6/vdzhup/2.py [hunk 11]
```diff
@@ -0,0 +1,7 @@
+while True:
+    user_password = input("Please enter the password: ")
+    if user_password == "First":
+        print("Congratulations! You have entered the correct password.")
+        break
+    else:
+        print("You entered the wrong password. Try it again.")
```
Comment (kolyasalubov) [inline HW6/vdzhup/2.py:7]: good job

File: HW4/vdzhup/1.py (hunk 6 above)
Comment (VolodymyrDzhupyna) [reply HW4/vdzhup/1.py:14]: If user_factorial = 1 or user_factorial = 0 the result of factorial will be 1.

**Pull request merged**